python/requirements.txt

```diff
@@ -18,3 +18,4 @@ decorator
 prettytable
 objgraph
 gast
```

Comment:
gast>=0.3.3 ?

Reply:
Sure, but since it is minor issue and my CI is waiting, I will change in next PR and inform you.